ibis/tests/sql/snapshots/test_compiler/test_agg_filter/out.sql

```diff
@@ -10,6 +10,6 @@ t1 AS (
 SELECT t1.*
 FROM t1
 WHERE t1.`a` = (
-  SELECT max(t1.`a`) AS `blah`
+  SELECT max(t1.`a`) AS `Max(a)`
```

Comment:
The `Alias` is no longer pushed down into the subquery, but that should not be necessary.

Reply:
Yep, the alias isn't needed here.